contrib/statesync.bash

```diff
@@ -2,38 +2,43 @@
 # microtick and bitcanna contributed significantly here.
 set -uxe
 
-# set environment variables
+# Set Golang environment variables.
 export GOPATH=~/go
 export PATH=$PATH:~/go/bin
 
-
-# Install Gaia
+# Install Gaia.
 make install
 
-# MAKE HOME FOLDER AND GET GENESIS
+# Initialize chain.
 gaiad init test 
-wget -O ~/.gaia/config/genesis.json https://cloudflare-ipfs.com/ipfs/Qmc54DreioPpPDUdJW6bBTYUKepmcPsscfqsfFcFmTaVig
 
-INTERVAL=1000
+# Prepare genesis file for cosmoshub-4.
```

Comment:
If you pass --chain-id cosmoshub-4 on `init` then downloading the genesis can be eliminated

Reply:
nice catch @glnro , this is because with state sync there's nothing about needing initial state to be accurate within the genesis file, right?

I haven't confirmed this personally, could I get one more ACK that this change could go in from @alexanderbez or @marbar3778 ?

Reply:
Thank you for the suggestion @glnro! I've tested it locally and it works well. I've pushed a fix that removes the explicit genesis file downloading.